integration_tests/run_tests.py

```diff
@@ -3,16 +3,22 @@
 import os
 import sys
 
-tests = [
-    "expr_01.py",
-    "expr_02.py",
+CUR_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__)))
+skip_list = [
+    "run_tests.py"
 ]
 
 def main():
+    tests = []
+    for file in os.listdir(CUR_DIR):
+        if file in skip_list:
+            continue
+        if file.endswith(".py") and not file.startswith("py_"):
+            tests.append(file)
```

Comment:
This works now, but it will not work once we add modules --- as we only want to call LPython on the "main" program, not the modules it imports. So we should explicitly list all the tests that we want to execute by hand, and that way we can have other Python files that can be imported, but they will not be directly tested. 

If you agree, I would leave the original explicit list of tests in there.

Reply:
Okay, agreed!!